### ISSUE TITLE
Game stages not actually unlocking anything

### ISSUE THREAD

**Issue by MudkipNinja** [jaredlll08/RecipeStages#53]
As the title says, I have a zs script in my scripts folder that only has the following:
`mods.recipestages.Recipes.setRecipeStage("hoe", "minecraft:stone_hoe");`

After a reload in the game, sure enough the recipe for the stone hoe is successfully removed and requires the gamestage "hoe"

The issue comes in when I grant a player the gamestage (using commands, scripts, or gamestage books), nothing seems to happen. The chat says "You have unlocked gamestage "hoe"!" but the recipe is still hidden and is also uncraftable. Plus I an continually grant myself the gamestage over and over.

I have linked my CT Log but I couldn't find any errors in there. My other zs scripts seem to be working fine as well.

Minecraft Version: 1.12.2
Forge Version: 14.23.5.2847
GameStages Version: 1.12.2-1.0.2
Craft Tweaker Version: 1.12-4.1.20.586

Note, I have been unable to replicate this on a clean install; just the 2 mods with forge seem to work fine, and the recipe unlocks. Its only in my modpack with 239 mods that it seems unable to work, which led me to believe it was a mod conflict, but I have been unable to find any.

Log: [https://pastebin.com/edRQe4Lj](url)

**Comment by jaredlll08** [comment]
As you said, it can't be reproduced with just recipe stages and gamestages, this means that there is nothing I can actually debug.

You need to find which mod is causing the actual issue and report it to them (or report to Gamestages).

**Comment by MudkipNinja** [comment]
Yeah I figured as much. I guess I was hoping for a "this version has issues with Tinkers" or some hail mary like that. I will work on debugging it with other mods slowly. I can post if I find it. Thank you!

**Comment by MudkipNinja** [comment]
So it turned out there was some weird call with a couple of mods. I know its not something that can be fixed here (I think) but I still wanted to post in case other people ran into this issue.

Apparently, when removing certain vanilla recipes, the Quark mod has alternative recipes that the game tries to use. After disabling Quark, the Recipestages worked fine. But Just Quark and Recipestages/Gamestages on its own *also* worked fine. So It turned out that another mod I was using, "Better with Mods", has a compatibility module that pulls any recipes that Quark adds and modifies them. Even extra recipes that modify vanilla tools. When both mods were used together it seemed that gamestages was trying to lock the recipe for a "Stone Hoe" in general, but when the player received the gamestage, it didn't know where to look to unlock it, because the quark recipe modified the vanilla one, and the quark one was modified further by better with mods.

The tl;dr solution was to disable the compatibility module between Quark and Better with Mods, and it seems to be running fine now.

**Comment by jaredlll08** [comment]
Closing since there is literally nothing I can do to fix a **bad** interaction from BWM
